devito/operator/profiling.py

```diff
@@ -514,8 +510,9 @@ def create_profile(name):
     if profiler.initialized:
         return profiler
     else:
-        warning("Couldn't set up `%s` profiler; reverting to `advanced`" % level)
-        profiler = profiler_registry['basic'](name)
```

Comment:
Here, it was reverting to `basic`

Reply:
ok but no need for that extra var imho (nitpicking)